packages/framework/src/Model/Product/Search/Export/ProductSearchExportWithFilterRepository.php

```diff
@@ -171,12 +172,33 @@ protected function extractResult(Product $product, int $domainId, string $locale
             'calculated_selling_denied' => $product->getCalculatedSellingDenied(),
             'selling_denied' => $product->isSellingDenied(),
             'availability' => $product->getCalculatedAvailability()->getName($locale),
-            'main_variant' => $product->isMainVariant(),
+            'is_main_variant' => $product->isMainVariant(),
             'detail_url' => $detailUrl,
             'visibility' => $visibility,
+            'uuid' => $product->getUuid(),
+            'unit' => $product->getUnit()->getName($locale),
+            'is_using_stock' => $product->isUsingStock(),
+            'stock_quantity' => $product->getStockQuantity(),
+            'variants' => $variantIds,
+            'main_variant' => $product->isVariant() ? $product->getMainVariant() : null,
```

Comment:
I'll need to add this thing to upgrade notes because it might break the migrate structure and it's needed to delete the structure and then create it again...